airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -9,7 +9,7 @@
     "properties": {
       "access_token": {
         "type": "string",
-        "description": "Log into Github and then generate a <a href=\"https://github.com/settings/tokens\"> personal access token</a>.",
+        "description": "Log into Github and then generate a <a href=\"https://github.com/settings/tokens\"> personal access token</a>. Separate multiple tokens with \",\"",
```

Comment:
Ideally, `type: string` would be an array of strings too. However for backwards compatibility this is not great. 

Could you create an issue to update this to a list of strings once #5396 is resolved?